src/wp-includes/classicpress/class-wp-compat.php

```diff
@@ -0,0 +1,310 @@
+<?php
+/**
+ * ClassicPress polyfills for blocks.
+ *
+ * @package ClassicPress
+ * @since CP-2.0.0
+ */
+
+class WP_Compat {
+
+	public $blocks_compatibility_level = null;
+
+	public function __construct() {
+
+		if ( $this->blocks_compatibility_level === null ) {
```

Comment:
Should be aiming for Yoda expressions - so `null === $this->blocks_compatibility_level`